src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -74,7 +74,7 @@ export const SuggestionSnippet: React.FC<Props> = ({
         ChangeSet.isNodeChange(suggestion) ||
         ChangeSet.isNodeAttrChange(suggestion)
       ) {
-        newSnippet = handleNodeChange(suggestion, view.state)
+        newSnippet = handleNodeChange(suggestion, view.state, getState().files)
```

Comment:
hm, why not just pass files right from the store without getState() and introduce it as a dependency on the useEffect?

Reply:
No reason 🤦‍♀️